libsrc/grabber/x11/X11Grabber.cpp

```diff
@@ -19,16 +19,16 @@ X11Grabber::X11Grabber(bool useXGetImage, int cropLeft, int cropRight, int cropT
 	_dstFormat(nullptr),
 	_srcPicture(None),
 	_dstPicture(None),
+	_horizontalDecimation(horizontalPixelDecimation),
+	_verticalDecimation(verticalPixelDecimation),
 	_screenWidth(0),
 	_screenHeight(0),
 	_croppedWidth(0),
 	_croppedHeight(0),
 	_image(0,0),
 	_log(Logger::getInstance("X11GRABBER"))
```

Comment:
big thx!
one small thing, can you fix coding style of initializer list according to coding style:
https://github.com/hyperion-project/hyperion.ng/blob/master/CODING_STYLE.md